circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/RenameTableOperation.java

```diff
@@ -40,20 +40,27 @@ public RenameTableOperation(DropTableService dropTableService) {
    * renamed.
    */
   public void execute(CloseableMetaStoreClient client, Table from, Table to) throws Exception {
-    LOG
-        .info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(),
-            to.getTableName());
+    LOG.info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(), to.getTableName());
     Table fromTable = client.getTable(from.getDbName(), from.getTableName());
     Table toTable = client.getTable(to.getDbName(), to.getTableName());
+    String fromDatabaseName = fromTable.getDbName();
     String fromTableName = fromTable.getTableName();
     String toTableName = toTable.getTableName();
     String toDelete = toTableName + DELETE_ME;
     try {
+      fromTable.setDbName(toTable.getDbName());
       fromTable.setTableName(toTableName);
       toTable.setTableName(toDelete);
+      LOG
+          .info("Alter table {}.{} to {}.{}", toTable.getDbName(), toTableName, toTable.getDbName(),
```

Comment:
```suggestion
          .info("Renaming table {}.{} to {}.{}", toTable.getDbName(), toTableName, toTable.getDbName(),
```

Reply:
Also, shouldn't the first two arguments be the "from" values? Same in the log statement below.

Reply:
Alternatively we remove these two new log statements and just keep the one we already had.

Reply:
It's more clear when you actually read the log messages in test and when running CT.